docs/source/notebooks.ipynb

```diff
@@ -0,0 +1,396 @@
+    "| [`--force-nb`](#force)                           | Overwrite the file if a notebook already exists.             |\n",
+    "| [`--disable-nb-exec`](#exec)                     | Prevent the generated notebook from being executed.          |\n",
+    "| [`--template-nb <path>`](#template)              | Use a custom Jupyter Notebook as template.                   |\n",
+    "| [`--nb-kernel <kernel>`](#kernel)                | Specify which kernel should be used to execute the notebook. |\n",
+    "\n",
+    "(notebook)=\n",
+    "\n",
+    "### `--notebook <path>` / `--nb <path>`\n",
+    "\n",
+    "Main argument to generate a notebook by providing the desired output path. A new _\\*.ipynb_ file will be created but if one exists, the user gets prompted to know if it should be kept or overwritten (answer by typing _yes_ or _no_). The default action can be configured with [`--update-nb`](#update), or [`--force-nb`](#force).\n",
+    "\n",
+    "(update)=\n",
+    "\n",
+    "### `--update-nb`\n",
+    "\n",
+    "This option defines the behavior when the provided path for the notebook points to an existing notebook. Instead of asking what to do, NPF will try to replace the data in the notebook with the most recent measurements. This is done, by replacing the cells that have the _data_ tag with `data = [...]` where `[...]` contains the results. No code will be lost, as long as no changes were made to any cell that has the _data_ tag.\n",
+    "\n",
+    "(force)=\n",
+    "\n",
+    "### `--force-nb`\n",
+    "\n",
+    "This option defines the behavior when the provided path for the notebook points to an existing notebook. This is a way to force NPF to overwrite the existing file with a newly created one. Changes made to the existing file will be lost forever with this operation. Make sure the file doesn't contain any important changes.\n",
+    "\n",
+    ":::{warning}\n",
+    "Any existing notebook at the given path will be overwritten and data may be lost.\n",
+    ":::\n",
+    "\n",
+    "(exec)=\n",
+    "\n",
+    "### `--disable-nb-exec`\n",
+    "\n",
+    "By default, the generated notebook gets executed making it ready for analysis by providing cell outputs and displaying graphs. This behavior can be disabled using this parameter. The resulting notebook now has to be executed by the user.\n",
+    "\n",
+    "(template)=\n",
+    "\n",
+    "### `--template-nb <path>`\n",
+    "\n",
+    "A template is used and converted into the final notebook generated by NPF. This template contains code for many types of graphs. With this parameter, you can provide a new template that will be used instead. The default path is located in the NPF project at [_npf/types/notebook/template.ipynb_](https://github.com/tbarbette/npf/blob/430612f2ceedaabc5697daad37624ac7e2cfb725/npf/types/notebook/template.ipynb) and this file can be used as starter for a custom one.\n",
```

Comment:
This link doesn't work yet. It should work once the pull request of the code gets accepted.